server/index.js

```diff
@@ -0,0 +1,67 @@
+const { PrismaClient } = require('@prisma/client');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const bcrypt = require('bcrypt');
+
+const prisma = new PrismaClient();
+const saltRounds = 10;
+const app = express();
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.post('/api/user/signup', async (req, res) => {
+  const {
+    username, password, email
+  } = req.body;
+  if (!username || !password || !email) {
+    return res.status(400).json({ message: 'Invalid Payload' });
+  }
+
+  const emailExists = await prisma.user.findUnique({
+    where: {
+      email
+    }
+  });
+
+  if (emailExists) {
+    res.status(409).json({
+      message: 'email already in use'
+    });
+
+    return;
+  }
+
+  const usernameExists = await prisma.user.findUnique({
+    where: {
+      username
+    }
+  });
+
+  if (usernameExists) {
+    res.status(409).json({
+      message: 'username already in use'
+    });
+  } else {
```

Comment:
Adding a `return` statement instead of an `else` would really help with readability.